wire-ios-sync-engine/Tests/Source/E2EI/SelfClientCertificateProviderTests.swift

```diff
@@ -141,7 +141,7 @@ class SelfClientCertificateProviderTests: MessagingTest {
 
 }
 
-private extension String {
+extension String {
```

Comment:
Actually this whole extension can be removed, there is the exact same code in `String+Mock.swift`.